adoc/SA_vars.adoc

```diff
@@ -17,9 +18,13 @@ endif::[]
 
 // could be "Azure","AWS","GCE" or "libvirt"
 //:cloud: Azure
-:cloud: AWS
+//:cloud: AWS
+:cloud: GCP
```

Comment:
Please don't check in such changes of the SA_vars.adoc file, as it would create a endless loop of changes due to switching clouds.

Reply:
Is there another way to create a GCP PDF version?